torch/onnx/utils.py

```diff
@@ -42,7 +42,7 @@ def set_training(model, mode):
 
 
 def export(model, args, f, export_params=True, verbose=False, training=False,
-           input_names=None, output_names=None, aten=False, export_raw_ir=False):
+           input_names=None, output_names=None, operator_export_type=OperatorExportTypes.ONNX):
```

Comment:
This changes the public interface? Did we advertise the bool args previously? If yes, add bc compat code to translate them to enum values